src/shared/lib/isSampleProfiled.ts

```diff
@@ -81,3 +82,24 @@ export function isSampleProfiledInMultiple(
         );
     }
 }
+
+export interface IGenePanelDataByProfileIdAndSample {
+    [profileId: string]: {
+        [sampleId: string]: GenePanelData;
+    };
+}
+
+export function isSampleProfiledInProfile(
+    genePanelMap: IGenePanelDataByProfileIdAndSample,
+    profileId: string | undefined,
+    sampleId: string | undefined
+) {
+    return (
```

Comment:
@gblaih i decided it was better to spell things out after all.  But at least just do it once place